diff --git a/config.def.h b/config.def.h
index 0e01717..0c9c2a0 100644
--- a/config.def.h
+++ b/config.def.h
@@ -82,33 +82,36 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/*
+ * Railscasts scheme by Ryan Bates (http://railscasts.com)
+ * Base16 st template by:
+ * Honza Pokorny <me@honza.ca>
+ * Daniel Mulford (github.com/dgmulf)
+ */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+    /* 8 normal colors */
+	[0] = "#2b2b2b", /* black   */
+	[1] = "#da4939", /* red     */
+	[2] = "#a5c261", /* green   */
+	[3] = "#ffc66d", /* yellow  */
+	[4] = "#6d9cbe", /* blue    */
+	[5] = "#b6b3eb", /* magenta */
+	[6] = "#519f50", /* cyan    */
+	[7] = "#e6e1dc", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8]  = "#5a647e", /* black   */
+	[9]  = "#da4939", /* red     */
+	[10] = "#a5c261", /* green   */
+	[11] = "#ffc66d", /* yellow  */
+	[12] = "#6d9cbe", /* blue    */
+	[13] = "#b6b3eb", /* magenta */
+	[14] = "#519f50", /* cyan    */
+	[15] = "#f9f7f3", /* white   */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
@@ -118,8 +121,8 @@ static const char *colorname[] = {
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultcs = 7;
+static unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
