diff --git a/config.def.h b/config.def.h
index 0e01717..0c9c2a0 100644
--- a/config.def.h
+++ b/config.def.h
@@ -82,33 +82,36 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/*
+ * Horizon Light scheme by Michaël Ball (http://github.com/michael-ball/)
+ * Base16 st template by:
+ * Honza Pokorny <me@honza.ca>
+ * Daniel Mulford (github.com/dgmulf)
+ */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+    /* 8 normal colors */
+	[0] = "#FDF0ED", /* black   */
+	[1] = "#E95678", /* red     */
+	[2] = "#29D398", /* green   */
+	[3] = "#FADAD1", /* yellow  */
+	[4] = "#26BBD9", /* blue    */
+	[5] = "#EE64AC", /* magenta */
+	[6] = "#59E1E3", /* cyan    */
+	[7] = "#403C3D", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8]  = "#BDB3B1", /* black   */
+	[9]  = "#E95678", /* red     */
+	[10] = "#29D398", /* green   */
+	[11] = "#FADAD1", /* yellow  */
+	[12] = "#26BBD9", /* blue    */
+	[13] = "#EE64AC", /* magenta */
+	[14] = "#59E1E3", /* cyan    */
+	[15] = "#201C1D", /* white   */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
@@ -118,8 +121,8 @@ static const char *colorname[] = {
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultcs = 7;
+static unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
