diff --git a/config.def.h b/config.def.h
index 0e01717..0c9c2a0 100644
--- a/config.def.h
+++ b/config.def.h
@@ -82,33 +82,36 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/*
+ * Apathy scheme by Jannik Siebert (https://github.com/janniks)
+ * Base16 st template by:
+ * Honza Pokorny <me@honza.ca>
+ * Daniel Mulford (github.com/dgmulf)
+ */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+    /* 8 normal colors */
+	[0] = "#031A16", /* black   */
+	[1] = "#3E9688", /* red     */
+	[2] = "#883E96", /* green   */
+	[3] = "#3E4C96", /* yellow  */
+	[4] = "#96883E", /* blue    */
+	[5] = "#4C963E", /* magenta */
+	[6] = "#963E4C", /* cyan    */
+	[7] = "#81B5AC", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8]  = "#2B685E", /* black   */
+	[9]  = "#3E9688", /* red     */
+	[10] = "#883E96", /* green   */
+	[11] = "#3E4C96", /* yellow  */
+	[12] = "#96883E", /* blue    */
+	[13] = "#4C963E", /* magenta */
+	[14] = "#963E4C", /* cyan    */
+	[15] = "#D2E7E4", /* white   */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
@@ -118,8 +121,8 @@ static const char *colorname[] = {
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultcs = 7;
+static unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
